test/run.sh

```diff
@@ -155,6 +155,17 @@ test_builder_node_version() {
   fi
 }
 
+test_nss_wrapper() {
+  local run_cmd="echo 'danbev:x:1000:1000:danbev test:/home/danbev:/bin/false' > passwd && LD_PRELOAD=libnss_wrapper.so NSS_WRAPPER_PASSWD=passwd NSS_WRAPPER_GROUP=group getent passwd danbev"
```

Comment:
Is the proxy `group` file not required?

Reply:
I'll take a closer look and see if this needs to be added to. I've tested this (though it does not work properly with regards to the error handling) with an image that does not have the `nss_wrapper` and it does produce the same error message as reported. But I'll follow up on this tomorrow. Just wanted to create this PR so that we can see how to manually test this with our production image.

Reply:
@danbev Thanks for helping out with this!